wagtail/wagtailadmin/wagtail_hooks.py

```diff
@@ -8,6 +8,7 @@
 from wagtail.wagtailadmin.navigation import get_explorable_root_page
 from wagtail.wagtailadmin.search import SearchArea
 from wagtail.wagtailadmin.utils import user_has_any_page_permission
+from wagtail.wagtailadmin.viewsets import viewsets
```

Comment:
Maybe it would be clearer to be able to import the `viewsets` instance directly from `wagtail.wagtailadmin`?
Or rename `viewsets` to `viewset_registry`, something like that.